### ISSUE TITLE
Not process duplicate frames?

### ISSUE THREAD

**Issue by SwayStar123** [xinntao/Real-ESRGAN#444]
Currently the exe file just runs through every frame and upscales it, even the duplicate frames, i feel like it would be alot faster if it accounted for duplicates

**Comment by xinntao** [comment]
A good idea~  Will update when we are free~

**Comment by TechnoMasterBoy** [comment]
Another thing that could speed up the process is to not upscale completely black/white frames. Simply resize them.
Though I feel like duplicate frames would give significantly bigger boost in speed due to hentai/anime usually having thousands of identical frames.

**Comment by SwayStar123** [comment]
Yes, I feel like with some simple python the whole process could be automated too, I dont think a user needs to seperate the video into frames and then merge it back themselves

**Comment by Splendide-Imaginarius** [comment]
Looks like https://github.com/xinntao/Real-ESRGAN/pull/252 would help here.

**Comment by TechnoMasterBoy** [comment]
> Looks like #252 would help here.

Now to wait 100 years for a windows build with it.
